File: py/Aksesoris.py
from PetShop import PetShop

class Aksesoris(PetShop): # inherit dari class PetShop
    # atribut
    __jenis = ""
    __bahan = ""
    __warna = ""
    
    # constructor
    def __init__(self, jenis="", bahan="", warna=""):
        self.__jenis = jenis
        self.__bahan = bahan
        self.__warna = warna
        
    # getter
    def getJenis(self): return self.__jenis
    def getBahan(self): return self.__bahan
    def getWarna(self): return self.__warna
    
    # setter
    def setJenis(self, jenis): self.__jenis = jenis
    def setBahan(self, bahan): self.__bahan = bahan
    def setWarna(self, warna): self.__warna = warna

File: java/Aksesoris.java
public class Aksesoris extends PetShop{ // inherit dari class PetShop
    // atribut
    private String jenis;
    private String bahan;
    private String warna;

    // constructor
    public Aksesoris(){
        
    }
    public Aksesoris(String jenis, String bahan, String warna){
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    // getter
    public String getJenis() { return this.jenis; }
    public String getBahan() { return this.bahan; }
    public String getWarna() { return this.warna; }

    // setter
    public void setJenis(String jenis) { this.jenis = jenis; }
    public void setBahan(String bahan) { this.bahan = bahan; }
    public void setWarna(String warna) { this.warna = warna; }



}